seaworthy/dockerhelper.py

```diff
@@ -7,6 +7,19 @@
 log = logging.getLogger(__name__)
 
 
+def fetch_missing_images(client, images):
```

Comment:
I kind of preferred just `fetch_images`... also not sure if we ever want to fetch a list of images, rather than just `fetch_image` for one image... and if we did _that_ would it make sense to return the image object?

Reply:
I'll change it back to `fetch_images()`. I thought `fetch_missing_images()` was a little more explicit, but there isn't really that much difference.

I think if we ever want to fetch and return an image, we can do that separately. This function is intended to ensure that whatever images we need are available when we need them without having to check and pull each time.